src/Dots.tsx

```diff
@@ -47,6 +47,12 @@ const Dots = ({
     childrenArr
   );
   const currentSlides = lookupTable[currentSlide];
+  const shouldRemoveDots = numberOfDotsToShow <= 1;
```

Comment:
There is a utility called notEnoughChildren in https://github.com/YIZHUANG/react-multi-carousel/blob/master/src/utils/common.ts.  This will better handle this.